OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -427,3 +472,50 @@ struct OptimobileArguments: Decodable {
         credentials = OptimobileCredentials(apiKey: apiKey, secretKey: secretKey)
     }
 }
+
+struct PreferenceCenterArguments: Decodable {
+    enum Error: Foundation.LocalizedError {
+        case emptyBase64
+        case failedDecodingBase64(String)
+
+        var errorDescription: String? {
+            switch self {
+            case .emptyBase64:
+                return "The base64 string is empty"
+            case let .failedDecodingBase64(string):
+                return "Failed on decoding base64 the value \(string)"
+            }
+        }
+    }
+
+    let version: String
+    let environment: String //TODO: Change this to enum
```

Comment:
in android it doesnt even do any mapping -- it just uses in url whatever is in the string. this is perhaps a little more flexible -- we dont have hardcoded PC regions in the sdk and we can e.g. easily update dev-pb --> dev